.github/workflows/tests.yaml

```diff
@@ -100,7 +100,7 @@ jobs:
   tests-matrix:
     # Use this matrix with multiple jobs defined in a reusable workflow:
     uses: ./.github/workflows/.tests-matrix.yaml
-    name: ${{ matrix.os.name }} (Python ${{ matrix.python-version }})
+    name: "${{ matrix.os.name }} (Python ${{ matrix.python-version }}) ${{ runner.arch }}"
```

Comment:
```suggestion
    name: "${{ matrix.os.name }} ${{ runner.arch }} (Python ${{ matrix.python-version }})"
```

Reply:
As I commented above, `runner` context is not available for `job.name`